src/main/kotlin/racingcar/Main.kt

```diff
@@ -1,14 +1,11 @@
 package racingcar
 
 fun main() {
-    val inputView = InputView()
-    inputView.excute()
-
     val race = Race()
 
-    race.start(inputView.carCount, inputView.roundCount)
+    val result = race.result(InputView.getCarNames(), InputView.getRoundCount())
 
-    val resultView = ResultView(race.roundList)
+    val resultView = ResultView(result)
     resultView.title()
     resultView.printResult()
```

Comment:
```suggestion
    ResultView(result).run {
        title()
        printResult()
    }
```
범위 연산자를 활용하면, 기능 단위로 묶을 수 있어요.